LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 Xorima
```

Comment:
FYI Sous-Chefs use the Apache-2.0 license https://github.com/sous-chefs/ruby_rbenv/blob/master/LICENSE